echopype/tests/mask/test_mask.py

```diff
@@ -1240,4 +1241,146 @@ def test_apply_mask_channel_variation(source_has_ch, mask, truth_da):
                                                 )
         
         # Check mask to match truth
+        print(masked_ds[var_name])
+        print(truth_da)
         assert masked_ds[var_name].equals(truth_da)
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "range_var, use_multi_channel_mask", [
+        ("echo_range", True),
+        ("echo_range", False),
+        ("depth", True),
+        ("depth", False),
+    ]
+)
+def test_apply_mask_dims_using_MVBS(range_var, use_multi_channel_mask):
+    """
+    Check for correct values and dimensions when using `apply_mask` to apply
+    frequency differencing masks to MVBS.
+    """
+    # Parse Raw File
+    ed = ep.open_raw(
+        raw_file="echopype/test_data/ek60/DY1801_EK60-D20180211-T164025.raw",
+        sonar_model="EK60"
+    )
+
+    # Compute Sv
+    ds_Sv = ep.calibrate.compute_Sv(ed)
+
+    if range_var == "depth":
+        # Add depth
+        ds_Sv = ep.consolidate.add_depth(ds_Sv)
+
+    # Compute MVBS
+    MVBS = ep.commongrid.compute_MVBS(
+        ds_Sv=ds_Sv,
+        range_var=range_var,
+        range_bin="1m",
+        ping_time_bin="10s"
+    )
+
+    if use_multi_channel_mask:
+        # Create two frequency differencing masks that in total cover channels
+        # 18kHz - 200kHz:
+        # (120kHz - 38kHz) and (38kHz - 18kHz)
+        diff = [
+            ep.mask.frequency_differencing(
+                source_Sv=MVBS,
+                freqABEq="120000.0Hz - 38000.0Hz > 0dB",
+                storage_options={},
+            ).expand_dims(dim={"channel": MVBS["channel"][0:2].data}),
+            ep.mask.frequency_differencing(
+                source_Sv=MVBS,
+                freqABEq="38000.0Hz - 18000.0Hz > 0dB",
+                storage_options={},
+            ).expand_dims(dim={"channel": MVBS["channel"][2:6].data}),
+        ]
```

Comment:
How about taking out the dependency on `ep.mask.frequency_differencing` here and just create a random bit map for the test? This way if `frequency_differencing` fails it would not affect this test. Same with the else condition.

Reply:
Oh that's clever I'll add this suggestion